src/structs/API.ts

```diff
@@ -0,0 +1,314 @@
+import fetch, { Headers } from 'node-fetch'
+import APIError from '../utils/APIError'
```

Comment:
here's an interesting example on casing. this would become a_p_i_error in snakecase